test/powershell/Host/HostUtilities.Tests.ps1

```diff
@@ -49,21 +74,26 @@ Describe "InvokeOnRunspace method as nested command" -tags "Feature" {
 Describe "InvokeOnRunspace method on remote runspace" -tags "Feature" {
     
     BeforeAll {
-        $wc = [System.Management.Automation.Runspaces.WSManConnectionInfo]::new()
-        $remoteRunspace = [runspacefactory]::CreateRunspace($host, $wc)
-        $remoteRunspace.Open()
+        $script:skipTest = $true
+        It "Get remote runspace for test" -Skip:(!$IsWindows) {
```

Comment:
I am getting mixed directions about this so I asked around and the consensus is that It should not be in a BeforeAll block.  The reason it is there is because it does try/catch and allows a meaningful message to be displayed.  However, I will change this so that it no longer resides in the BeforeAll block.  Another issue with this is skipping tests because of a configuration failure.  Skip is not the right thing to use because it has a different meaning, i.e., tests that are not intended to run on a platform or configuration.  So I am also going to remove the skip and let individual tests fail because of the invalid configuration.  This way all tests affected by the configuration failure will be reported.